test-apps/testpurchasesuiandroidcompatibility/build.gradle.kts

```diff
@@ -48,10 +49,29 @@ android {
     }
 }
 
+emerge {
+    // TODO: RevenueCat to set from CircleCi variables
+    apiToken.set(System.getenv("EMERGE_API_TOKEN"))
+
+    vcs {
+        // TODO: RevenueCat to set from CircleCi variables
+        sha.set("")
```

Comment:
I think it might be good to get this from environment variables too. CircleCI sets this as `CIRCLE_SHA1`, but we can pass a more generic env like `COMMIT_SHA1` or something like that (or default to empty if none found). Wdyt?

Reply:
Just needs to match GitHub for proper status check posting. Updated to use this variable!